backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -306,6 +313,10 @@ export class TransactionLinkResolver {
       } finally {
         releaseLock()
       }
+      // trigger to send transaction via dlt-connector
+      sendTransactionsToDltConnector().catch(() => {
```

Comment:
This function `sendTransactionsToDltConnector` should also write the database


```
          const dltTx = DltTransaction.create()
          dltTx.transactionId = transaction.id
          await DltTransaction.save(dltTx)
```

You could do something like `sendTransactionsToDltConnector(dbTransaction)` this way its encapsulated.

Also consider to hook to the events.
Have the client be triggered by events (e.g. sendCoin Event) instead

Reply:
That was my first idea too, to write the dlt_transaction-entry in this separated and async function, but i prefer here the fact to simply insert a new dlt-entry with the transactions.id in the frame of the transactions transaction. Because if the transactions transaction will be not commited, then the dlt-entry should also not exist. The async sendTransactionsToDltConnector searches only for new and still not successfully transferred dlt-entries. 

The idea to lean on hooking like events seems good, i will do some researches there.

Reply:
When you call the function after queryRunner.commitTransaction() like I original did, you know that the transaction was written into db. Because if an error happen, code after queryRunner.commitTransaction() won't be called. Instead it will jump to the catch block.

Reply:
shifted the dlt-transaction creation outside the transactions-writing transaction in the async send-method